src/wallet/components/WalletDisconnect.tsx

```diff
@@ -0,0 +1,28 @@
+import { useConnect, useDisconnect } from 'wagmi';
+import { cn, pressable, text } from '../../styles/theme';
+import { disconnectSvg } from './disconnectSvg';
+import { useCallback } from 'react';
+
+export function WalletDisconnect() {
```

Comment:
Let's have this as `WalletDropdownDisconnet` so it makes sense this is an Item inside the Dropdown.

Otherwise it feels like a random button I can fit anywhere even outside the dropdownn.